submissions/RemoveBG_Extension/readme.md

```diff
@@ -0,0 +1,14 @@
+# CopyBG
+A tool for copying transparent images
+
+## What it does
+It is a tool that is on the right-click menu. When you right click an image, it shows a custom option with our logo. If you click it, it'll display a popup, which shows whether the copied image is sucessful or not. If it is, the image with the removed backgruond will be copied to your clipboard. This will allow  you to paste it anywhere. Additionally, it will display a link on the popup that links to the image with the removed background.
+
+## How to use
+1. Clone this repo with `git clone https://github.com/B-Eddie/RemoveBG_Extension`
+2. Create an account from (remove.bg)[https://accounts.kaleido.ai/users]
+3. Create an API key
```

Comment:
Ok i just realised I need to create and include an API key in the extension. 

I suggest adding these instructions within the extension and allowing the user to include their api key within it rather than having to include it in the source code.